### PR TITLE
docs: update docs for 5.2

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

File: doc/assets/toolkit-wizard.png
Comment (LProulx86) [inline doc/assets/toolkit-wizard.png:1]: What difference between doc/assets/toolkit-wizard.png and doc/assets/material-toolkit-wizard-feature.png ?

Comment (kazo0) [reply doc/assets/toolkit-wizard.png:1]: The app name :)

Comment (LProulx86) [reply doc/assets/toolkit-wizard.png:1]: Ah ! yes :)

File: doc/cupertino-getting-started.md [hunk 1]
```diff
@@ -1,19 +1,67 @@
...
+# Uno Cupertino Toolkit Library
 
-The Uno.Toolkit.Cupertino library is available as NuGet packages that can be added to any new or existing Uno solution.
+The Uno Cupertino Toolkit library is available as NuGet packages that can be added to any new or existing Uno solution.
 
 ## Installation
 
-> [!NOTE]
-> As of [Uno Platform 4.7](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/), the solution template of the Uno app has changed. There is no longer a Shared project (.shproj), it has been replaced with a regular cross-platform library containing all user code files, referred to as the **App Code Library** project. This also implies that package references can be included in a single location without the previous need to include those in all project heads.
+### Creating a new project with the Uno Material Toolkit
```
Comment (agneszitte) [inline doc/cupertino-getting-started.md:10]: Would be nice I think @kazo0  to point to the get started doc around the first step here:

With a note like in some other pages of the doc with:

```
> [!NOTE]
> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
```

File: doc/cupertino-getting-started.md [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
+dotnet new unoapp -o CupertinoToolkitApp -toolkit -theme cupertino
+```
+
+---
+
+### Installing Uno Cupertino Toolkit in an existing project
+
+Depending on the type of project template that the Uno Platform application was created with, follow the instructions below to install the Uno Cupertino Toolkit.
+
+# [**Single Project Template**](#tab/singleproj)
```
Comment (agneszitte) [inline doc/cupertino-getting-started.md:30]:
```suggestion
#### [**Single Project Template**](#tab/singleproj)
```

File: doc/cupertino-getting-started.md [hunk 3]
```diff
@@ -1,19 +1,67 @@
...
+
+                <!-- Load Cupertino Toolkit resources -->
+                <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
+                <CupertinoToolkitResources xmlns="using:Uno.Toolkit.UI.Cupertino" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Application.Resources>
+    ```
+
+# [**Multi-Head Project Template**](#tab/multihead)
```
Comment (agneszitte) [inline doc/cupertino-getting-started.md:60]:
```suggestion
# [**Multi-Head Project Template (Legacy)**](#tab/multihead)
```

File: doc/cupertino-getting-started.md [hunk 4]
```diff
@@ -28,29 +76,23 @@ The Uno.Toolkit.Cupertino library is available as NuGet packages that can be add
...
-            <!-- ... -->
-
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
 
-### Installing the Cupertino Toolkit on previous versions of Uno Platform
-
-If your application is based on the older solution template that includes a shared project (.shproj), follow these steps:
+# [**Shared Project (.shproj) Template**](#tab/shproj)
```
Comment (agneszitte) [inline doc/cupertino-getting-started.md:83]:
```suggestion
#### [**Shared Project (.shproj) Template (Legacy)**](#tab/shproj)
```

File: doc/getting-started.md [hunk 5]
```diff
@@ -28,41 +25,87 @@ This article is a guide for installing the base Uno.Toolkit library, additional
 
 ### Installation
 
-1. Open an existing Uno project, or create a new Uno project using the `Multi-Platform App (Uno Platform)` template.
-2. In the Solution Explorer panel, right-click on your app's **App Code Library** project (`PROJECT_NAME.csproj`) and select `Manage NuGet Packages...`
-3. Install the [**`Uno.Toolkit.WinUI`**](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material) package
-4. Add the resources to `AppResources.xaml`:
+### Creating a new project with Uno Toolkit
+
+# [**Wizard**](#tab/wizard)
```
Comment (agneszitte) [inline doc/getting-started.md:30]:
```suggestion
#### [**Wizard**](#tab/wizard)
```

File: doc/getting-started.md [hunk 6]
```diff
@@ -28,41 +25,87 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+
+1. Follow the steps in the [Getting Started with Visual Studio](xref:Uno.GettingStarted.CreateAnApp.VS2022#create-the-app) instructions to launch the Uno Platform Template Wizard.
+2. Select `Toolkit` under the `Features` section.
+
+    ![Toolkit selection in the Uno Platform Template Wizard](assets/toolkit-wizard.png)
+
+# [**CLI**](#tab/cli)
```
Comment (agneszitte) [inline doc/getting-started.md:37]:
```suggestion
#### [**CLI**](#tab/cli)
```

File: doc/getting-started.md [hunk 7]
```diff
@@ -28,41 +25,87 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+dotnet new unoapp -o ToolkitApp -toolkit
+```
+
+---
+
+### Installing Uno Toolkit in an existing project
+
+Depending on the type of project template that the Uno Platform application was created with, follow the instructions below to install Uno Toolkit.
+
+# [**Single Project Template**](#tab/singleproj)
```
Comment (agneszitte) [inline doc/getting-started.md:57]:
```suggestion
#### [**Single Project Template**](#tab/singleproj)
```

File: doc/getting-started.md [hunk 8]
```diff
@@ -28,41 +25,87 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+
+                <!-- Code ommitted of brevity -->
+
+                <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Application.Resources>
+    ```
+
+# [**Multi-Head Project Template**](#tab/multihead)
```
Comment (agneszitte) [inline doc/getting-started.md:80]:
```suggestion
# [**Multi-Head Project Template (Legacy)**](#tab/multihead)
```

File: doc/getting-started.md [hunk 9]
```diff
@@ -28,41 +25,87 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
-            <!-- ... -->
-
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
 
-#### Previous Installation Method
-
-If your application is based on the older solution template that includes a shared project (.shproj), follow these steps:
+# [**Shared Project (.shproj) Template**](#tab/shproj)
```
Comment (agneszitte) [inline doc/getting-started.md:96]:
```suggestion
# [**Shared Project (.shproj) Template (Legacy)**](#tab/shproj)
```

File: doc/material-getting-started.md [hunk 10]
```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
 
 ## Installation
 
-> [!NOTE]
-> As of [Uno Platform 4.7](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/), the solution template of the Uno app has changed. There is no longer a Shared project (.shproj), it has been replaced with a regular cross-platform library containing all user code files, referred to as the **App Code Library** project. This also implies that package references can be included in a single location without the previous need to include those in all project heads.
+### Creating a new project with the Uno Material Toolkit
```
Comment (agneszitte) [inline doc/material-getting-started.md:37]: Would be nice I think @kazo0  to point to the get started doc around the first step here:

With a note like in some other pages of the doc with:

```
> [!NOTE]
> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
```

File: doc/material-getting-started.md [hunk 11]
```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
...
+
+# [**Wizard**](#tab/wizard)
```
Comment (agneszitte) [inline doc/material-getting-started.md:39]:
```suggestion
#### [**Wizard**](#tab/wizard)
```

File: doc/material-getting-started.md [hunk 12]
```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
...
+1. Follow the steps in the [Getting Started with Visual Studio](xref:Uno.GettingStarted.CreateAnApp.VS2022#create-the-app) instructions to launch the Uno Platform Template Wizard.
+2. Select `Material` under the `Theme` section.
+
+    ![Material selection in the Uno Platform Template Wizard](assets/material-toolkit-wizard-theme.png)
+
+3. Select `Toolkit` under the `Features` section.
+
+    ![Toolkit selection in the Uno Platform Template Wizard](assets/material-toolkit-wizard-feature.png)
+
+# [**CLI**](#tab/cli)
```
Comment (agneszitte) [inline doc/material-getting-started.md:50]:
```suggestion
#### [**CLI**](#tab/cli)
```

File: doc/material-getting-started.md [hunk 13]
```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
...
+
+### Installing Uno Material Toolkit in an existing project
+
+Depending on the type of project template that the Uno Platform application was created with, follow the instructions below to install the Uno Material Toolkit.
 
-1. Open an existing Uno project, or create a new Uno project using the `Multi-Platform App (Uno Platform)` template.
-2. In the Solution Explorer panel, right-click on your app's **App Code Library** project (`PROJECT_NAME.csproj`) and select `Manage NuGet Packages...`
-3. Install the [**`Uno.Toolkit.WinUI.Material`**](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material) package
-4. Add the resources to `AppResources.xaml`:
+# [**Single Project Template**](#tab/singleproj)
```
Comment (agneszitte) [inline doc/material-getting-started.md:70]:
```suggestion
#### [**Single Project Template**](#tab/singleproj)
```

File: doc/material-getting-started.md [hunk 14]
```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
...
+
+                <!-- Code ommitted of brevity -->
+
+                <MaterialToolkitTheme xmlns="using:Uno.Toolkit.UI.Material" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Application.Resources>
+    ```
+
+# [**Multi-Head Project Template**](#tab/multihead)
```
Comment (agneszitte) [inline doc/material-getting-started.md:93]:
```suggestion
#### [**Multi-Head Project Template (Legacy)**](#tab/multihead)
```

File: doc/material-getting-started.md [hunk 15]
```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
...
-            <!-- ... -->
-
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
 
-### Installing the Material Toolkit on previous versions of Uno Platform
-
-If your application is based on the older solution template that includes a shared project (.shproj), follow these steps:
+# [**Shared Project (.shproj) Template**](#tab/shproj)
```
Comment (agneszitte) [inline doc/material-getting-started.md:109]:
```suggestion
#### [**Shared Project (.shproj) Template (Legacy)**](#tab/shproj)
```

File: doc/material-getting-started.md [hunk 16]
```diff
@@ -94,38 +140,40 @@ If your application is based on the older solution template that includes a shar
...
 
 ## Using C# Markup
 
-The Material Toolkit library also has support for C# Markup through a [Uno.Toolkit.WinUI.Material.Markup](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material.Markup) NuGet Package.
+The Uno Material Toolkit library also has support for C# Markup through a [Uno.Toolkit.WinUI.Material.Markup](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material.Markup) NuGet Package.
 
-To get started with Material Toolkit in your C# Markup application, add the `Uno.Toolkit.WinUI.Material.Markup` NuGet package to your **App Code Library** project and your platform heads.
-Then, add the following code to your `AppResources.cs`:
+To get started with the Uno Material Toolkit in your C# Markup application, add the `Uno.Toolkit.WinUI.Material.Markup` NuGet package to your application project.
+Then, add the following code to your `App.cs`:
```
Comment (jeromelaban) [inline doc/material-getting-started.md:176]:
```suggestion
Then, add the following code to your `App.xaml.cs`:
```

File: doc/material-getting-started.md [hunk 17]
```diff
@@ -139,7 +187,7 @@ this.Build(r => r.UseMaterialToolkit(
 ```
 
 > [!NOTE]
-> The [Uno.Toolkit.WinUI.Material.Markup](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material.Markup) NuGet package includes the base [Toolkit Markup package](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Markup) as a dependency. Therefore, there is no need to add the `Uno.Toolkit.WinUI.Markup` package separately. Furthermore, the `UseMaterialToolkit` extension method also initializes the Toolkit library, so there is no need to call the `UseToolkit` extension method in your `AppResources.cs`.
+> The [Uno.Toolkit.WinUI.Material.Markup](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material.Markup) NuGet package includes the base [Uno Toolkit Markup package](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Markup) as a dependency. Therefore, there is no need to add the `Uno.Toolkit.WinUI.Markup` package separately. Furthermore, the `UseMaterialToolkit` extension method also initializes the base Uno Toolkit library, so there is no need to call the `UseToolkit` extension method in your `App.cs`.
```
Comment (jeromelaban) [inline doc/material-getting-started.md:190]:
```suggestion
> The [Uno.Toolkit.WinUI.Material.Markup](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material.Markup) NuGet package includes the base [Uno Toolkit Markup package](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Markup) as a dependency. Therefore, there is no need to add the `Uno.Toolkit.WinUI.Markup` package separately. Furthermore, the `UseMaterialToolkit` extension method also initializes the base Uno Toolkit library, so there is no need to call the `UseToolkit` extension method in your `App.xaml.cs`.
```

**Review by jeromelaban — APPROVED** [review]

**Comment by kazo0** [comment]
@mergifyio backport release/stable/6.0

File: doc/material-getting-started.md [hunk 18]
```diff
@@ -35,43 +35,93 @@ Initialization of the Material Toolkit resources is handled by the specialized `
...
+    ```
+
+2. Initialize `MaterialToolkitTheme` in the `App.xaml`:
+
+    ```xml
+    <Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+
+                <!-- Code ommitted of brevity -->
```
Comment (agneszitte) [inline doc/material-getting-started.md:88]:
```suggestion
                <!-- Code omitted of brevity -->
```

File: doc/getting-started.md [hunk 19]
```diff
@@ -28,41 +25,87 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+    ```
+
+2. Initialize `ToolkitResources` in the `App.xaml`:
+
+    ```xml
+    <Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+
+                <!-- Code ommitted of brevity -->
```
Comment (agneszitte) [inline doc/getting-started.md:72]:
```suggestion
                <!-- Code omitted of brevity -->
```

File: doc/cupertino-getting-started.md [hunk 20]
```diff
@@ -1,19 +1,70 @@
...
+    ```
+
+2. Initialize the resources in the `App.xaml`:
+
+    ```xml
+    <Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+
+                <!-- Code ommitted of brevity -->
```
Comment (agneszitte) [inline doc/cupertino-getting-started.md:48]:
```suggestion
                <!-- Code omitted of brevity -->
```

**Review by agneszitte — APPROVED** [review]

**Pull request merged**